Data/Sys/GameSettings/GZ2.ini

```diff
@@ -17,3 +17,7 @@ ImmediateXFBEnable = False
 
 [Video_Enhancements]
 ArbitraryMipmapDetection = True
+EFBScaledExclude = 200
+EFBScaledExcludeBlur = True
+EFBScaledExcludeAlt = True
+EFBScaledExcludeEnabled = True
```

Comment:
All other places were listed under "[Video_Settings]" but these ones

Reply:
Thanks for checking